addon/components/bs4/bs-form/element/label.js

```diff
@@ -3,7 +3,7 @@ import FormElementLabel from 'ember-bootstrap/components/base/bs-form/element/la
 export default FormElementLabel.extend({
   tagName: 'label',
 
-  classNames: ['form-control-label'],
+  classNames: ['col-form-label'],
```

Comment:
You're right about the BS docs.

My testing and looking into the BS source indicates that `.col-form-label` is the way going forward because it handles both vertical and horizontal forms, for margins, padding and validations. For inline forms, it is not an issue because the `.form-inline` class on the form handles this.

If this isn't the case then I'll update accordingly.